cilium/cmd/endpoint_get.go

```diff
@@ -28,6 +30,54 @@ import (
 
 var lbls []string
 
+// L4Policy is nested JSON, so search 'result' for strings that have
+// backslashes inside, and interpet the JSON.
+func expandNestedJSON(result bytes.Buffer) bytes.Buffer {
```

Comment:
So by doing this it won't actually be pure JSON. Can you add a flag to (disabled by default) to output pure JSON?

Reply:
Done in the first commit.

Reply:
I'm so used watching that output that I didn't realized is pure JSON :)